test/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { ChartConfig, ChartConfigProps } from "charts/ChartConfig"
+
+import * as fixtures from "./fixtures"
+
+export function createConfig(props?: Partial<ChartConfigProps>) {
+    const config = new ChartConfig(new ChartConfigProps(props))
+    // ensureValidConfig() is only run on non-node environments, so we have
+    // to manually trigger it.
+    config.ensureValidConfig()
+    return config
+}
+
+export function setupChart(
+    id: 677 | 792,
+    varId: 3512 | 104402,
```

Comment:
It was `number`, but I always look up which IDs we have in the `fixtures` folder, and I thought it's more convenient if VSCode can just show that in the tooltip as you're writing the function. Can definitely change it to `number` if there are downsides!

Reply:
Alright! No, I can't think of any immediate downsides, just that we have to keep this in sync (but since TS will complain otherwise, that should be a given)